tests/validate-bom/action.yml

```diff
@@ -44,4 +44,19 @@ runs:
     run: |
       echo "working directory: $(pwd)"
       mvn -ntp -B install
```

Comment:
Is `install` needed anymore, or could the test be based solely on `dependency:tree`?

Reply:
Install may detect something. (I don’t think of a case now) At least it should succeed and I don’t consider it’s adding a additional complexity.